test/populate_test/bdl_commands_test/circulation_loop_test.py

```diff
@@ -23,22 +23,35 @@
     Zone,
     BDL_ZoneCWValveOptions,
 )
+from rpd_generator.bdl_structure.bdl_commands.project import (
+    RunPeriod,
+    Holidays,
+    BDL_RunPeriodKeywords,
+    BDL_HolidayKeywords,
+    BDL_HolidayTypes,
+)
 from rpd_generator.config import Config
+from rpd_generator.artifacts.ruleset_project_description import (
+    RulesetProjectDescription,
+)
 from rpd_generator.artifacts.ruleset_model_description import RulesetModelDescription
 from rpd_generator.bdl_structure.bdl_commands.circulation_loop import *
-from rpd_generator.utilities import schedule_funcs
```

Comment:
@bradcline-kc instead of using schedule_funcs, use RunPeriod and Holidays objects in the test setUp as done here.